staticdata/achievements.js

```diff
@@ -86,7 +86,7 @@ module.exports = [
         name: 'Snap your fingers',
         desc: 'Reset collection for the first time',
         actions: ['col', 'collection'],
-        check: (ctx, user) => user.completedcols.length > 0,
+        check: (ctx, user) => user.completedcols.map(x => x.amount).sort((a, b) => b.amount - a.amount)[0] > 0,
```

Comment:
Does this work? It seems like you do the map extracting the amount which is just an integer. And after that you sort it by non-existing property. Maybe just `sort((a, b) => b - a)`?

Reply:
Yeah it works, x=> x.amount is a holdover from another thing I was testing. It works with or without map. Just leave this PR open and I'll give it without map in a day or three. Want to finish the other things I am working on before pushing again.

Reply:
I can just remove it myself. It just seems like sort is compering undefined values